order/models.py

```diff
@@ -0,0 +1,40 @@
+from django.db      import models
+from user.models    import Users
+from product.models import Products
+
+class Payments(models.Model):
+    user       = models.models.ForeignKey(Users, on_delete=models.CASCADE)
+    card       = models.CharField(max_length=50)
+    createtime = models.DateField(auto_now_add=True)
+
+class Orders(models.Model):
+    status     = models.ForeignKey("Status", on_delete=models.CASCADE)
+    payment    = models.ForeignKey("Payments", on_delete=models.CASCADE)
+    address    = models.ForeignKey("Address", on_delete=models.CASCADE)
```

Comment:
누구의 주문인지 어떻게 알 수 있나요?
order에 user_id를 참조할 수 있도록 수정해주세요~
payment 테이블에서 order_id를 참조하는게 맞습니다. 
```suggestion
class Orders(models.Model):
    user = models.Foreign...
    status     = models.ForeignKey("Status", on_delete=models.CASCADE)
    address    = models.ForeignKey("Address", on_delete=models.CASCADE)
    created_at
    updated_at
```